### PR TITLE
update external pr kpis

### PR DESCRIPTION
**Pull request by PaliC**
Updates external contribution kpis to only show the pr count. Furthermore, we now have a monthly view
Screenshot 2023-03-31 at 3 46 11 PM

### CODE REVIEW

File: .github/workflows/stronghold.yml [hunk 1]
```diff
@@ -25,7 +25,6 @@ jobs:
             --base-commit=${{ github.event.pull_request.base.sha }} \
             --head-commit=${{ github.event.pull_request.head.sha }}
 
-      docker-image: buildpack-deps:scm
```
Comment (kit1980) [inline .github/workflows/stronghold.yml:28]: Why is this change here?

**Review by kit1980 — APPROVED** [review]
LGTM, but it looks like the PR has couple of a unintended changes to stronghold files?

File: torchci/rockset/pytorch_dev_infra_kpis/monthly_contribution_stats.lambda.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "default_parameters": [
+    {
+      "name": "granularity",
+      "type": "string",
+      "value": "month"
+    },
+    {
+      "name": "startTime",
+      "type": "string",
+      "value": "\t2022-05-01T00:06:32.839Z"
```
Comment (kit1980) [inline torchci/rockset/pytorch_dev_infra_kpis/monthly_contribution_stats.lambda.json:12]: Do you need  `\t` here?

Comment (PaliC) [reply torchci/rockset/pytorch_dev_infra_kpis/monthly_contribution_stats.lambda.json:12]: idk, the automated script willed it.
